### ISSUE TITLE
Web: OpendetailViewInNewTab opens default page

### ISSUE THREAD

**Issue by emeyke** [eXpandFramework/eXpand#198]

**Comment by apobekiaris** [comment]
Until next milestone you can use the next controller.

```cs
    public class OpenDetailViewInNewTabController:Xpand.ExpressApp.Web.SystemModule.OpenDetailViewInNewTabController {
        protected override string GetViewId(object currentObject) {
            return Application.FindDetailViewId(currentObject, View);
        }
    }

```

**Comment by apobekiaris** [comment]
It should also be available in the next nightly see #197

**Comment by apobekiaris** [comment]
Closing issue for age. Feel free to reopen it at any time.

.Thank you for your contribution.
